.circleci/config.yml

```diff
@@ -62,6 +62,7 @@ jobs:
   unit_test:
     docker:
       - image: circleci/node:7.10
+    resource_class: large
```

Comment:
It looks like this change has already been made in v1.3.2 here (https://github.com/yext/answers/commit/83843468000227ba9e273f345c6dbbb81dddbc59). Can you merge v1.3.2 into this branch so this PR only shows changes for the accordion card? Otherwise LGTM!